Core/Frameworks/Flake/Core/Collection.php

```diff
@@ -31,11 +31,11 @@ class Collection extends \Flake\Core\FLObject implements \Iterator {
     protected $aCollection = [];
     protected $aMeta = [];
 
-    function current() {
+    function current(): mixed {
```

Comment:
I thought that this is a new feature in PHP 8. Will it "work" OK in PHP 7?

Reply:
Hmm, right.

With `mixed`, php 7 complains:

> Uncaught TypeError: Return value of x() must be an instance of mixed, [...] returned

Without `mixed`, php 8.1 complains:

> PHP Fatal error: During inheritance of Iterator: Uncaught ErrorException: Return type of Flake\\Core\\Collection::key() should either be compatible with Iterator::key(): mixed, or the #[\\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

So we should probably just suppress it to still keep php 7 compatibility for a while.